lib/src/internal/http/descope_client.dart

```diff
@@ -41,18 +41,22 @@ class DescopeClient extends HttpClient {
     });
   }
 
-  Future<MaskedAddressServerResponse> otpUpdateEmail(String email, String loginId, String refreshJwt) {
+  Future<MaskedAddressServerResponse> otpUpdateEmail(String email, String loginId, String refreshJwt, UpdateOptions? options) {
     return post('auth/otp/update/email', MaskedAddressServerResponse.decoder, headers: authorization(refreshJwt), body: {
       'loginId': loginId,
       'email': email,
+      "addToLoginIDs": options?.addToLoginIds,
```

Comment:
Mind the double quotes (vs single above)

Reply:
Good catch

Reply:
Done